godtools/Managers/TractManager.swift

```diff
@@ -16,57 +16,64 @@ class TractManager: NSObject {
+                    let primaryTextColorString: String = (manifest!.element?.attribute(by: "primary-text-color")?.text) ?? "rgba(59, 164, 219, 1)"
+                    let textColorString: String = (manifest!.element?.attribute(by: "text-color")?.text) ?? "rgba(90, 90, 90, 1)"
+                    
+                    for child in manifest!["pages"].children {
+                        if child.element?.name == "page" {
+                            let page = loadPage(child)
+                            pages.append(page)
+                        }
+                    }
+                    
+                    tractColors.primaryColor = primaryColorString.getRGBAColor()
+                    tractColors.primaryTextColor = primaryTextColorString.getRGBAColor()
+                    tractColors.textColor = textColorString.getRGBAColor()
+                }
             }
         }
         
-        let tractColors = TractColors()
-        tractColors.primaryColor = primaryColorString.getRGBAColor()
-        tractColors.primaryTextColor = primaryTextColorString.getRGBAColor()
-        tractColors.textColor = textColorString.getRGBAColor()
-        
         return (pages, tractColors)
     }
     
     func loadPage(_ child: XMLIndexer) -> XMLIndexer{
-        let resource = child.element?.attribute(by: "id")?.text
+        let resource = child.element?.attribute(by: "src")?.text
         let page = loadXMLFile(resource!)
-        return page
+        return page!
     }
     
-    func loadXMLFile(_ resource: String) -> XMLIndexer {
+    func loadXMLFile(_ resourcePath: String) -> XMLIndexer? {
+        let documentsPath = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)[0]
+        let file = documentsPath.appending("/Resources/").appending(resourcePath)
+        
         var xml: XMLIndexer?
-        if let filepath = Bundle.main.path(forResource: resource, ofType: "xml") {
-            do {
-                let content = try String(contentsOfFile: filepath)
-                
-                let regex = try! NSRegularExpression(pattern: "\n", options: NSRegularExpression.Options.caseInsensitive)
-                let range = NSMakeRange(0, content.characters.count)
-                let modString = regex.stringByReplacingMatches(in: content, options: [], range: range, withTemplate: "")
-                
-                xml = SWXMLHash.parse(modString.condenseWhitespace())
-            }
-            catch {
-                // error
-            }
-        } else {
-            // error
+        do {
+            let content = try String(contentsOfFile: file, encoding: String.Encoding.utf8)
+            
+            let regex = try! NSRegularExpression(pattern: "\n", options: NSRegularExpression.Options.caseInsensitive)
+            let range = NSMakeRange(0, content.characters.count)
+            let modString = regex.stringByReplacingMatches(in: content, options: [], range: range, withTemplate: "")
```

Comment:
More than that, the new lines are being printed on the labels, so that affects the display of the content on the UI.